modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgFilterToSQL.java

```diff
@@ -71,10 +71,7 @@ public String escapeName(String name) {
                         + ",'utc')"; // utc -- everything must be consistent -- see
                 // literal visitor
             } else if (java.sql.Date.class.isAssignableFrom(binding)) {
-                return "date("
-                        + super_result
-                        + ",'utc')"; // utc -- everything must be consistent -- see
-                // literal visitor
+                return "date(" + super_result + ")";
```

Comment:
Javadoc is out of sync with this functionality, still describes use of `utc`.

Reply:
Q: Do you need to do the same thing for Timestamp.class above?

Reply:
no, timezone handling of Timestamp is correct.  Java Date "sort of" has a timezone.

Reply:
Thanks for the discussion, I have updated the conversation above.